diagnostic.md

```diff
@@ -91,9 +112,7 @@ div {
 }
 ```
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+The color will be yellow and the font-family will be sans-serif
```

Comment:
Actually, the font-family will be cursive. Do you remember the concept of specificity that Rachel talked about? Take a look at the HTML again.